src/js/edu-benefits/components/NavButtons.jsx

```diff
@@ -0,0 +1,157 @@
+
+    const goBack = () => this.props.onNavigate(previousPath);
+    const goForward = () => this.handleContinue(nextPath);
+
+    const backButton = (
+      <ProgressButton
+          onButtonClick={goBack}
+          buttonText="Back"
+          buttonClass="usa-button-outline"
+          beforeText="«"/>
+    );
+
+    const nextButton = (
+      <ProgressButton
+          onButtonClick={goForward}
+          buttonText="Continue"
+          buttonClass="usa-button-primary"
+          afterText="»"/>
+    );
+
+    if (path === '/review-and-submit') {
+      let submitButton;
+      let submitMessage;
+
+      if (submission.status === false) {
+        submitButton = (
+          <ProgressButton
+              onButtonClick={this.handleSubmit}
+              buttonText="Submit Application"
+              buttonClass="usa-button-primary"/>
+        );
+      } else if (submission.status === 'submitPending') {
+        submitButton = (
+          <ProgressButton
+              onButtonClick={this.handleSubmit}
+              buttonText="Sending..."
+              buttonClass="usa-button-disabled"/>
+        );
+      } else if (submission.status === 'applicationSubmitted') {
+        submitButton = (
+          <ProgressButton
+              onButtonClick={this.handleSubmit}
+              buttonText="Submitted"
+              buttonClass="form-button-green"
+              beforeText="&#10003;"/>
+        );
+      } else {
+        submitMessage = (<div className="usa-alert usa-alert-error">
+          <p><strong>Due to a system error, we weren't able to process your application. Please try again later.</strong></p>
+          <p>We apologize for the inconvenience. If you'd like to complete this form by phone, please call 877-222-VETS (8387) and press 2, M-F 7:00 a.m.to 7:00 p.m. (CST), Sat 9:00 a.m. to 5:30 p.m. (CST).</p>
+        </div>);
+        submitButton = (
+          <ProgressButton
+              onButtonClick={this.handleSubmit}
+              buttonText="Send Failed"
+              buttonClass="usa-button-secondary form-button-disabled"
+              beforeText="x"/>
+        );
+      }
+
+      return (<div>
+        <div className="row progress-buttons">
+          <div className="small-6 medium-5 columns">
+            {backButton}
+          </div>
+          <div className="small-6 medium-5 columns">
+            {submitButton}
+          </div>
+          <div className="small-1 medium-1 end columns">
+            <div className={this.state ? 'spinner' : 'hidden'}>&nbsp;</div>
+          </div>
+        </div>
+        <div className="row">
+          <div className="medium-12 columns">
+          {submitMessage}
+          </div>
+        </div>
+      </div>);
+    } else if (path === '/submit-message') {
+      return (
+        <div className="row progress-buttons">
+          <div className="small-6 medium-5 columns">
+            {/* TODO: Figure out where this button should take the user. */}
```

Comment:
I think we can remove this TODO now, I think we decided it should take people back to the home page on vets.gov.